deployments/dev/environment.py

```diff
@@ -41,26 +41,122 @@ def env() -> Mapping[str, Optional[str]]:
+            'tdr:datarepo-dev-67240cf2:snapshot/hca_dev_f86f1ab41fbb4510ae353ffd752d4dfc__20210901_20210903:',
+            'tdr:datarepo-dev-5ef7f2e2:snapshot/hca_dev_455b46e6d8ea4611861ede720a562ada__20210901_20210903:',
+            'tdr:datarepo-dev-40283c27:snapshot/hca_dev_3a69470330844ece9abed935fd5f6748__20210901_20210903:',
+            'tdr:datarepo-dev-b3632667:snapshot/hca_dev_d3446f0c30f34a12b7c36af877c7bb2d__20210901_20210903:',
+            'tdr:datarepo-dev-279f1986:snapshot/hca_dev_5ee710d7e2d54fe2818d15f5e31dae32__20210901_20210903:',
+            'tdr:datarepo-dev-bdc9f342:snapshot/hca_dev_3e329187a9c448ec90e3cc45f7c2311c__20210901_20210903:',
+            'tdr:datarepo-dev-86c60513:snapshot/hca_dev_1cd1f41ff81a486ba05b66ec60f81dcf__20210901_20210903:',
+            'tdr:datarepo-dev-12b7a9e1:snapshot/hca_dev_7880637a35a14047b422b5eac2a2a358__20210901_20210903:',
+            'tdr:datarepo-dev-4cf05ce2:snapshot/hca_dev_2a72a4e566b2405abb7c1e463e8febb0__20210901_20210903:',
+            'tdr:datarepo-dev-71926fdc:snapshot/hca_dev_c893cb575c9f4f26931221b85be84313__20210901_20210903:',
+            'tdr:datarepo-dev-e8e0a59a:snapshot/hca_dev_f8aa201c4ff145a4890e840d63459ca2__20210901_20210903:',
+            'tdr:datarepo-dev-f31edbc2:snapshot/hca_dev_4d6f6c962a8343d88fe10f53bffd4674__20210901_20210903:',
+            'tdr:datarepo-dev-376d3f4a:snapshot/hca_dev_bd40033154b94fccbff66bb8b079ee1f__20210901_20210903:',
+            'tdr:datarepo-dev-38e08b5c:snapshot/hca_dev_08b794a0519c4516b184c583746254c5__20210901_20210903:',
+            'tdr:datarepo-dev-aba01389:snapshot/hca_dev_7b947aa243a74082afff222a3e3a4635__20210831_20210907:',
+            'tdr:datarepo-dev-24e672db:snapshot/hca_dev_cddab57b68684be4806f395ed9dd635a__20210831_20210907:',
+            'tdr:datarepo-dev-6fdac3db:snapshot/hca_dev_e8808cc84ca0409680f2bba73600cba6__20210902_20210907:',
+            'tdr:datarepo-dev-0c5c20b5:snapshot/hca_dev_8c3c290ddfff4553886854ce45f4ba7f__20210902_20210907:',
+            'tdr:datarepo-dev-e2ab8487:snapshot/hca_dev_03c6fce7789e4e78a27a664d562bb738__20210902_20210907:',
+            'tdr:datarepo-dev-b839d6c7:snapshot/hca_dev_e526d91dcf3a44cb80c5fd7676b55a1d__20210902_20210907:',
+            'tdr:datarepo-dev-311340f6:snapshot/hca_dev_2d8460958a334f3c97d4585bafac13b4__20210902_20210907:',
+            'tdr:datarepo-dev-4c3e6011:snapshot/hca_dev_248fcf0316c64a41b6ccaad4d894ca42__20210907_20210907:',
+            'tdr:datarepo-dev-76de829d:snapshot/hca_dev_2043c65a1cf84828a6569e247d4e64f1__20210831_20210907:',
+            'tdr:datarepo-dev-24e9529e:snapshot/hca_dev_f83165c5e2ea4d15a5cf33f3550bffde__20210901_20210908:',
+            'tdr:datarepo-dev-1dce87e5:snapshot/hca_dev_ae71be1dddd84feb9bed24c3ddb6e1ad__20210916_20210916:',
+            # Managed access:
+            'tdr:datarepo-dev-02c59b72:snapshot/hca_dev_99101928d9b14aafb759e97958ac7403__20210830_20210903:',
```

Comment:
Not consistent with sheet:

![image](https://user-images.githubusercontent.com/5143256/134227530-ca9bcc08-2044-457a-acc8-1609d431fbf1.png)